sope-mime/NGMime/NGMimeType.m

```diff
@@ -88,6 +88,7 @@ + (void)initialize {
   }
 }
 
+// TODO: this should be reimplemented with a lookup table
```

Comment:
It would be great to uniform the comments to use /* */ or //, I'd prefer the former...